index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <body>
+
+        <main id="main-doc">
+          <header>
+              <nav id="navbar">
+                  <a class="logo" href="#"><h1>Climate Change</h1></a> 
+                  <div class="nav-links">
+                      <ul>
+                          <li>
+                            <a href="#">Home</a>
+                          </li>
+                          <li>
+                              <a href="#">Region</a> 
+                          </li>
+                          <li>
+                              <a href="#">Newsletter</a> 
+                          </li>
+                          <li>
+                              <a href="#">About Us</a> 
+                          </li>
+                          <li>
+                              <a href="#">Contact Us</a>
+                          </li>
+              
+                      </ul>
+                  </div>
```

Comment:
would be adviseable to use double space indentation rather than using the tab key

Reply:
please kindly review your use of indentation. You appear to have used the tab key rather than double space as recommended on new lines and tags. 😉